### ISSUE TITLE
[GeoTransf] TypeError() 

### ISSUE THREAD

**Issue by laurabenzan** [fmonegaglia/pyris#3]
Dear Federico, 

Whilst running PyRIS with all the image segmentation options, the following error was encountered:

<img width="555" alt="Pyris_TS_1" src="https://user-images.githubusercontent.com/32585554/81005078-48978a80-8e45-11ea-95ae-2b27de8a2af6.PNG">

I'd like to convert to str where needed however, I can't pin point exactly where it goes wrong or why - I can't determine if it's an input-relate issue or a tool-related issue.

The context of the job:
-Conda Environment with Py3.7
-10 Landsat images, in their respective directories as specified. 

Thanks in advance, 

Laura.-

**Comment by fmonegaglia** [comment]
Dear Laura,
I guess the problem relies in tge pytgon version. You are using version
3.x, while pyris was developed in version 2.7.
Unfortunately there is no retrocompatibility and the binary encoding of the
pickle package is different.
Good news is that python2.7 should come installed with Linux.
You just have to install pyris with python 2.7.
Before you do that, to be sure you run the correct one, i suggest you
remove the one you installed under py3, wgich you can find by running
"which pyris" on the linux terminal.
Also, conda is not really needed here, you just need the packages.
Let me know if this helps.
Best,
F

Il Lun 4 Mag 2020, 21:30 Laura B <notifications@github.com> ha scritto:

**Comment by laurabenzan** [comment]
Dear Federico, 

I see. OK! Thanks

I have to use conda since the uni lab has permissions for certain things. Anyway, I installed it again using a py2.7 environment. It's working ok but right now it's encountering an error that it's more system based (below). I referred it to ICT for them to do the necessary path referrals. 

<img width="689" alt="Pyris_TS_2" src="https://user-images.githubusercontent.com/32585554/81088859-7d621b00-8ef3-11ea-808b-45b56d61271c.PNG">

I'll keep you updated in order to close the thread. 

Laura.-
